manifests/o/Ombrelin/PandocGui/1.1.35/Ombrelin.PandocGui.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS0.5-1-22000-282
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Ombrelin.PandocGui
+PackageVersion: "1.1.35"
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /S
```

This is not the Silent install switch for Inno, did they change InstallerType?
```suggestion
InstallerSwitches:
  Silent: /S
```